dtfs-central-api/src/helpers/amendments/filter-completed-amendments.test.ts

```diff
@@ -0,0 +1,26 @@
+import { AMENDMENT_STATUS, TfmFacilityAmendment } from '@ukef/dtfs2-common';
+import { aTfmFacilityAmendment } from '../../../test-helpers';
+import { filterCompletedAmendments } from './filter-completed-amendments';
+
+describe('filter-completed-amendments', () => {
+  describe('filterCompletedAmendments', () => {
+    it('should return all completed amendments', () => {
```

Comment:
should we add a test where there are no completed amendments, and with and without?

Reply:
File was only temporarily in the diff between the base getting merged into main and main getting merged into this.